torch/distributions/beta.py

```diff
@@ -10,36 +10,36 @@
 
 class Beta(Distribution):
     r"""
-    Creates a Beta distribution parameterized by concentration `alpha` and `beta`.
+    Creates a Beta distribution parameterized by concentration `concentration1` and `concentration0`.
```

Comment:
nit: Could you wrap docstrings to <=80 characters so that `help(Beta)` works in jupyter notebooks?